src/targets/linux-uhid/main.c

```diff
@@ -23,52 +23,10 @@
 #include "platform/linux-uhid/uhid.h"
 #include "protocol/protocol.h"
 #include "protocol/reports.h"
+#include "util/hid_descriptors.h"
 
 #define OI_MOUSE_REPORT_ID 0x01
```

Comment:
you can use `MOUSE_REPORT_ID` instead, defined in the same "util/hid_descriptors.h"